script/generate-documentation

```diff
@@ -35,7 +34,7 @@ function update_docs {
 
         cd ..
         git add _includes/api.html
-        git add index.html current.pdf
+        git add docs/index.html docs/current.pdf
```

Comment:
Before it was a find for all the files, so you also need to add docs/api/testapi.html